daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/RestrictionUnion.scala

```diff
@@ -168,7 +168,7 @@ final class Restriction private (xmlArg: Node, val simpleTypeDef: SimpleTypeDefB
     if (hasPattern) {
       val lPattern = localBaseFacets.filter { case (f, v) => f == Facet.pattern }
       val rPattern = remoteBaseFacets.filter { case (f, v) => f == Facet.pattern }
-      val cPattern = lPattern.union(rPattern)
+      val cPattern = lPattern ++: rPattern
```

Comment:
I prefer named methods for set operations generally. So they removed 'union' ? Grrrrrr.

Reply:
They deprecated Seq.union but I believe Set.union is still available